bots/csr/src/test/java/org/openjdk/skara/bots/csr/CSRBotTests.java

```diff
@@ -162,13 +162,19 @@ void keepLabelForNotApprovedCSR(TestInfo testInfo) throws IOException {
             localRepo.push(editHash, repo.url(), "edit", true);
             var pr = credentials.createPullRequest(repo, "master", "edit", issue.id() + ": This is an issue");
 
+            // Run bot
+            TestBotRunner.runPeriodicItems(bot);
+
+            // The bot should have kept the CSR label
```

Comment:
Is this a check for if the bot added the label automatically?

Reply:
yes, I revised the comment for readability.